README.md

```diff
@@ -47,7 +47,8 @@ Find example applications that run on top of the Tock kernel written in both
 
 Read our [getting started guide](doc/Getting_Started.md) to get the correct
 version of the Rust compiler, then look through the `/kernel`, `/capsules`,
-`/chips`, and `/boards` directories.
+`/chips`, and `/boards` directories. There are also generated [source code
+docs](https://docs.tockos.org/kernel/).
```

Comment:
```suggestion
docs](https://docs.tockos.org).
```

Just link to the main page -- the lack of a landing page and the current redirect to `/kernel` is something I still do intend to fix someday